app/presenters/conviction_result_presenter.rb

```diff
@@ -3,6 +3,10 @@ def to_partial_path
     'results/conviction'
   end
 
+  def motoring?
```

Comment:
Let's move this to the decorator that was created for these situations, `ConvictionDecorator`, with a method along the lines:

```ruby
  def motoring?
    ConvictionType::ADULT_MOTORING.eql?(self)
  end
```

Then in the view, we use: `@presenter.conviction_type.motoring?`